src/vs/workbench/contrib/workspace/browser/workspace.contribution.ts

```diff
@@ -441,6 +447,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 					localize('workspace.trust.startupPrompt.once', "Ask for trust the first time an untrusted workspace is opened."),
 					localize('workspace.trust.startupPrompt.never', "Do not ask for trust when an untrusted workspace is opened."),
 				]
+			},
+			[WORKSPACE_TRUST_EMPTY_WORKSPACE]: {
+				type: 'boolean',
+				default: false,
+				included: !isWeb,
+				description: localize('workspace.trust.emptyWorkspace.description', "Controls whether or not the empty workspace is trusted within VS Code."),
```

Comment:
I think the description should mention something like "by default" since it can be changed after the fact per empty window